src/app/components/search/search.component.html

```diff
@@ -66,7 +66,10 @@
     <div class="inner-layout__content">
         <div class="inner-layout__panel">
             <div class="adf-search-results">
-                <adf-search-filter #searchFilter></adf-search-filter>
+                <adf-search-filter
+                    #searchFilter
+                    [ngClass]="{ 'adf-search-filter--hidden': !searchedWord }"></adf-search-filter>
```

Comment:
is there any reason for not using `*ngIf` in this case?

Reply:
I tried but found out that if I do that the SearchFilterComponent would be subscribing the 'queryBuilder.updated' event after it has been already emitted